test/rpc_test.py

```diff
@@ -1176,7 +1176,11 @@ def test_call_method_on_rref(self):
 
         self.assertEqual(result, sum(vals))
 
-    def _test_rref_leak(self, ignore_leak):
+    # Notice `rpc.api.shutdown()` accesses `_delete_all_user_rrefs`
+    # through `torch.distributed.rpc.api`, so patching
+    # `torch.distributed.rpc._delete_all_user_rrefs` will not help.
+    @mock.patch.object(torch.distributed.rpc.api, "_delete_all_user_rrefs")
+    def _test_rref_leak(self, mock_delete_all_user_rrefs, ignore_leak):
```

Comment:
`mock_delete_all_user_rrefs` isn't used anywhere in this PR?

Reply:
Right, the argument is added by the patch decorator. I can rename it to start with understore to indicate that it's a var name not used.